front/src/views/user/history-add-trip/history-add-trip.view.js

```diff
@@ -0,0 +1,53 @@
+import { MmtTextInput } from '@components';
+import { requiredValidationMixin } from '@mixins';
+import { mapGetters } from 'vuex';
+
+export default {
+  mixins: [
+    requiredValidationMixin
+  ],
+  components: {
+    MmtTextInput
+  },
+  data () {
+    return {
+      newTripValues: {
+        departureDate: this.$options.filters.date(new Date(), 'yyyy-MM-dd'),
+        arrivalDate: this.$options.filters.date(new Date(), 'yyyy-MM-dd'),
+        departureTime: this.$options.filters.date(new Date(), 'HH:mm'),
+        arrivalTime: this.$options.filters.date(new Date(), 'HH:mm'),
+        currency: 'USD',
+        departureCountryId: null,
+        arrivalCountryId: null
+      },
+      calendarLocale: {
+        'header-date-format': dt => this.$options.filters.date(dt, 'LLL yyyy'),
+        'month-format': dt => this.$options.filters.date(dt, 'LLL'),
+        'title-date-format': dt => this.$options.filters.date(dt, 'iiiiii LLL dd'),
+        'weekday-format': dt => this.$options.filters.date(dt, 'EEEEEE')
+      },
+      formValidity: false
+    };
+  },
+  computed: {
+    ...mapGetters('dictionaries', ['genders', 'countries', 'languages'])
+  },
+  methods: {
+    createNewTrip () {
+      this.$store.dispatch('trip/createNewTrip', {
+        departureCountryId: this.newTripValues.departureCountryId,
+        departureCity: this.newTripValues.departureCity,
+        departureDate: `${this.newTripValues.departureDate}T${this.newTripValues.departureTime}Z`,
+        arrivalCountryId: this.newTripValues.arrivalCountryId,
+        arrivalCity: this.newTripValues.arrivalCity,
+        arrivalDate: `${this.newTripValues.arrivalDate}T${this.newTripValues.arrivalTime}Z`,
+        totalPrice: +this.newTripValues.totalPrice,
+        currency: this.newTripValues.currency,
+        flightTime: this.newTripValues.flightTime,
+        transplantTime: this.newTripValues.transplantTime,
+        differenceInTime: this.newTripValues.differenceInTime
```

Comment:
you can try to use destruction
https://medium.com/@stasonmars/%D0%B4%D0%B5%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D1%83%D1%80%D0%B8%D0%B7%D0%B0%D1%86%D0%B8%D1%8F-%D0%B2-es6-%D0%BF%D0%BE%D0%BB%D0%BD%D0%BE%D0%B5-%D1%80%D1%83%D0%BA%D0%BE%D0%B2%D0%BE%D0%B4%D1%81%D1%82%D0%B2%D0%BE-b865bb71f376
https://learn.javascript.ru/destructuring-assignment